sonar-java-plugin/src/main/resources/static/documentation.md

```diff
@@ -63,6 +63,19 @@ Example: `sonar.java.source=1.6`
 
 If the property is provided, the analysis will take the source version into account, and execute related rules accordingly. At run time, each of these rules will be executed – or not – depending of the Java version used by sources within the project. For instance, on a correctly configured project built with Java 6, rules targeting Java 7 and Java 8 will never raise issues, even though they are enabled in the associated rule profile.
 
+## Skipping unchanged files
+Starting with SonarQube 9.4, and by default, the Java analyzer optimizes the analysis of unchanged files in pull requests.
```

Comment:
Is that documentation visible in SonarCloud? If yes, then we should find a way to remove "SonarQube 9.4".
Or say: "Starting from April 2022"

Reply:
It looks like we are sourcing the [sonarcloud documentation](https://docs.sonarcloud.io/advanced-setup/languages/java/)  from the same file. I will change the text to refer to April 2022 rather than a specific version.